selfdrive/loggerd/loggerd.cc

```diff
@@ -654,21 +656,14 @@ int main(int argc, char** argv) {
   while (!do_exit) {
     for (auto sock : poller->poll(100 * 1000)){
       while (true) {
-        Message * msg = sock->receive(true);
-        if (msg == NULL){
+        AlignedMessage amsg(sock->receive(true));
```

Comment:
only data from frame_sock  need to be parsed.alignedmesage is not suitable for this situation